config/sources/families/mvebu64.conf

```diff
@@ -1,6 +1,6 @@
 enable_extension "marvell-tools"
 ARCH=arm64
-BOOTBRANCH='branch:v2022.01'
+BOOTBRANCH='branch:v2022.01"
```

Comment:
it was correct before

Reply:
Oh, that was unintentional.  Yay review!